tables/tr-g2.tbl

```diff
@@ -0,0 +1,17 @@
+#+locale:tr
+#+type:literary
+#+contraction:full
+#+grade:2
+#+dots:8
+
+# TODO: Please correct the metadata above. It is not meant to be
```

Comment:
Actually I forgot to mention that ideally we want all the meta-data in the actual tables (also the `include braille-patterns.cti` part). Since you are in contact with the table author right now, could you ask him to verify and make correct any corrections if needed, and then put everything in tr-g2.ctb? Also the "official documentation about the implemented braille code" would be a nice extra :).